documentation/docs/requirements/use-case-descriptions.md

```diff
@@ -4,49 +4,49 @@ sidebar_position: 5
 
 # Use-Case Descriptions
 
-### Challenge Oriented (Gamer)
+### Use Case 1
 
-<i> As a User, I like to maximize the progression of in-game characters. I enjoy the process of assigning rewards to these characters and to see them grow.</i>
+<i> As a User, I want to use the resources I've collected in the application to progress my avatar. </i>
```

Comment:
"to progress my avatar" sounds kind of awkward. Perhaps "to enhance" or "improve"?